.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+ignore =
+    E402, # Ignore import not being at top of test
+    F631, # Assertion is always true in test_example.py
```

Comment:
There were some other formatting issues in the test module (quite probably there since my original commits).  Realizing this is mostly a comment about the necessity of the flake exception, here's a prospective set of changes that I think should make flake and pytest happier:

```
def test_import_getVersions_originalVersions():
    """
    Arrange: Load the primary module.
    Act: Retrieve the versions.
    Assert: Versions match the version numbers at the time of project creation.
    """
    assert (
        # fmt: off
        # '0.0.1' == {{cookiecutter.package_name}}.__version__,
        # fmt: on
        "This test is expected to fail when the version increments. "
        "It is here only as an example and you can remove it."
    )


@pytest.mark.parametrize("a,b,c", [(1, 2, 5), (3, 4, 25)])
def test_ab_addSquares_equalsC(a, b, c):
    """
    Arrange: Acquire the first two parameters (a and b).
    Act: Add the squares of the first two parameters (a and b).
    Assert: The sum of the squares equals the third parameter (c).

    :param a: the first parameter
    :param b: the second parameter
    :param c: the result of adding the squares of a and b
    """
    assert a * a + b * b == c, (
        "'c' should be the sum of the squares of 'a' and 'b'. "
        "This is an example test and can be removed.",
    )
```

Reply:
Okay, I will add this. Seems to be most direct fix.